drivers/qemu/driver.go

```diff
@@ -238,13 +242,7 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 		HealthDescription: drivers.DriverHealthy,
 	}
 
-	bin := "qemu-system-x86_64"
-	if runtime.GOOS == "windows" {
-		// On windows, the "qemu-system-x86_64" command does not respond to the
-		// version flag.
-		bin = "qemu-img"
-	}
-	outBytes, err := exec.Command(bin, "--version").Output()
+	outBytes, err := exec.Command("qemu-system-x86_64", "--version").Output()
```

Comment:
Do we know this will always be present even on non x86_64 environments? Should we have a plugin configuration lets you override the emulator binary used for fingerprinting?